tools/jil/server/asset-server.js

```diff
@@ -242,7 +246,7 @@ class AgentInjectorTransform extends AssetTransform {
           let rspData = rawContent
             .split('{loader}').join(tagify(disableSsl + loaderContent))
```

Comment:
Noticed: it likely has no practical effect since nothing's supposed to go wrong in the config part, but customers don't get script tags on each line right? Noticed that as I was looking at final `rspData`